daemon/algod/api/server/v2/account.go

```diff
@@ -430,9 +449,12 @@ func AssetParamsToAsset(creator string, idx basics.AssetIndex, params *basics.As
 		Total:         params.Total,
 		Decimals:      uint64(params.Decimals),
 		DefaultFrozen: &frozen,
-		Name:          strOrNil(params.AssetName),
-		UnitName:      strOrNil(params.UnitName),
-		Url:           strOrNil(params.URL),
+		Name:          strOrNil(sanitizePrintableUTF8String(params.AssetName)),
+		NameB64:       strOrNil(base64.StdEncoding.EncodeToString([]byte(params.AssetName))),
```

Comment:
No - we're *storing* one copy, but returning (sometimes) both, so that clients that use b64 could always decode that and could ignore the non-b64 version.